src/User/Persistence.hs

```diff
@@ -0,0 +1,8 @@
+module User.Persistence where
+
+import Snap.Snaplet.PostgresqlSimple
+import User.Query
+import User.Types
+
+selectAllUsers :: (HasPostgres m) => m [User]
+selectAllUsers = query_ allUsersQuery
```

Comment:
@kinorius123 Unfortunately, `postgresql-simple` don't assure us that the `User` type will be serialized correctly. Therefore, I think we should add a test for this query. I can give you a hand with that. But let me know what you think.

@sestrella @dbalseiro I'd like to know your opinion, too.

Reply:
@CristhianMotoche  @kinorius123 Yeap... I think we should start adding test coverage from the beginning... I was thinking about also addind an e2e test... WDYT?

Reply:
@dbalseiro Sounds good to me, too.